src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Properties/Resources.resx

```diff
@@ -396,4 +396,7 @@
   <data name="UpdatedResourceGroup" xml:space="preserve">
     <value>Updated resource group '{0}' in location '{1}'</value>
   </data>
+  <data name="OperationFailedWithTimeOut" xml:space="preserve">
+    <value>Operation failed because a request time out.</value>
```

Comment:
I am not saying it's not a word - I am just saying grammatically - "Operation failed because the request timed out"

---
In reply to: [66566451](https://github.com/Azure/azure-powershell/pull/2412#discussion_r66566451) [](ancestors = 66566451)